templates/property/display-generic.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 template:
+  deprecated: true
```

Comment:
why mark as deprecated rather than deleting this?

Reply:
Currently we are using the existing channels from the methodology repo. Marking this as deprecated to complete the rollout process then will come back and delete these channels when we are ready.